kuma/users/tests/test_views.py

```diff
@@ -46,11 +45,6 @@ def wiki_user_github_account(wiki_user):
     )
 
 
-@pytest.fixture
-def beta_testers_group(db):
```

Comment:
This is still used in this file. Does pytest automatically find it in the conftest file where you've moved it to?

Reply:
Yes, all of the test files within any given folder have access to the fixtures defined in `conftest.py` file within that folder as well as all folders upwards in the hierarchy.